tests-e2e/step-definitions/standard-ui/button.js

```diff
@@ -72,6 +72,19 @@ Then('I want to see a button {string}', async (content) => {
   client.useCss(); // switches back to css selector
 });
 
+/**
+ * Asserts that a button with a certain content is not visible.
+ */
+Then('I do not want to see a button {string}', async (content) => {
+  // xpath will be used as the locating strategy so all the selectors you pass should be xpath selectors
+  client.useXpath();
+  const xPathSelector = buttonSelector(content);
+
+  await client.expect.element(xPathSelector).to.not.be.present.after(5000);
```

Comment:
does this work without waiting 5 seconds? Is it the same as waitForElementNotPresent() ?

Reply:
Naturally it would work without the wait time, but then we can't know if it's due to slow render. It's just to be safe.